HMCL/src/main/resources/assets/lang/I18N_zh_CN.properties

```
@@ -1211,6 +1211,74 @@ sponsor.hmcl=Hello Minecraft! Launcher 是一个免费、自由、开放源代
 system.architecture=架构
 system.operating_system=操作系统
 
+terracotta=多人联机
+terracotta.easytier=关于 EasyTier
+terracotta.terracotta=Terracotta | 陶瓦联机
+terracotta.status=联机大厅
+terracotta.back=退出
+terracotta.network_warning=多人联机基于 p2p，最终联机体验和您的网络情况有较大关系。
+terracotta.sudo_installing=HMCL 需要验证您的密码才能安装联机核心
+terracotta.from_local.title=联机核心第三方下载链接
+terracotta.from_local.desc=在部分地区，联机核心默认下载渠道可能不稳定
+terracotta.from_local.guide=您应当下载名为 %s 的联机核心包。下载完成后，请将文件拖入当前界面来安装。
+terracotta.from_local.file_name_mismatch=您应当下载名为 %1$s 的联机核心包，而非 %2$s
+terracotta.status.bootstrap=正在收集信息
+terracotta.status.uninitialized.not_exist=未下载联机核心
+terracotta.status.uninitialized.not_exist.title=下载联机核心（约 8MB）
+terracotta.status.uninitialized.update=需更新联机核心
+terracotta.status.uninitialized.update.title=更新联机核心（约 8MB）
+terracotta.status.uninitialized.desc=您承诺，在多人联机全过程中，您将严格遵守您所在国家或地区的全部法律法规
+terracotta.confirm.title=用户须知
+terracotta.confirm.desc=多人联机基于 p2p，最终联机体验和您的网络情况有较大关系。您承诺，在多人联机全过程中，您将严格遵守您所在国家或地区的全部法律法规。
+terracotta.status.preparing=正在下载联机核心（请勿退出启动器）
+terracotta.status.launching=正在初始化联机核心
+terracotta.status.unknown=正在初始化联机核心
+terracotta.status.waiting=联机核心已就绪
+terracotta.status.waiting.host.title=我想当房主
+terracotta.status.waiting.host.desc=创建房间并生成邀请码，与好友一起畅玩
+terracotta.status.waiting.host.launch.title=您似乎忘记启动游戏了
+terracotta.status.waiting.host.launch.desc=未能找到正在运行的游戏
+terracotta.status.waiting.host.launch.skip=游戏已启动
+terracotta.status.waiting.guest.title=我想当房客
+terracotta.status.waiting.guest.desc=输入房主提供的邀请码加入游戏世界
+terracotta.status.waiting.guest.prompt.title=请输入房主提供的邀请码
+terracotta.status.waiting.guest.prompt.invalid=邀请码错误
+terracotta.status.scanning=正在扫描局域网世界
+terracotta.status.scanning.desc=请<a href="hmcl://game/launch">启动游戏</a>，进入单人存档，按下 ESC 键，选择对局域网开放，点击创建局域网世界。
+terracotta.status.scanning.back=这将同时停止扫描局域网世界。
+terracotta.status.host_starting=正在启动房间
+terracotta.status.host_starting.back=这将同时彻底关闭房间，其他房客将退出并不再能重新加入该房间。
+terracotta.status.host_ok=已启动房间
+terracotta.status.host_ok.code=邀请码（已自动复制到剪贴板）
+terracotta.status.host_ok.code.copy=复制邀请码
+terracotta.status.host_ok.code.desc=请提醒您的朋友在 HMCL 多人联机功能中选择房客模式，并输入该邀请码。
+terracotta.status.host_ok.back=这将同时彻底关闭房间，其他房客将退出并不再能重新加入该房间。
+terracotta.status.guest_starting=正在加入房间
+terracotta.status.guest_starting.back=这不会影响其他房客加入当前房间。
+terracotta.status.guest_ok=已加入房间
+terracotta.status.guest_ok.back=这不会影响其他房客加入当前房间。
```

```suggestion
terracotta.status.guest_ok.back=如果当前已有游戏连接到创建者的世界，这些连接将会中断。
```

我觉得这个地方不需要和创建者的文案对应，因为从常理上来讲加入者退出就是不应该影响其他加入者的。

---

这里确实不需要强调房客退出会影响其他玩家。但“如果当前已有游戏连接到创建者的世界”这句话有些奇怪，容我再想想。